app/src/main/java/com/android/doctorapp/ui/profile/ProfileViewModel.kt

```diff
@@ -168,6 +171,21 @@ class ProfileViewModel @Inject constructor(
         _navigationListener.value = R.id.action_user_profile_to_myDoctors
     }
 
+    fun clickedOnDarkTheme() {
+        if (isDarkThemeClicked.value == true) {
```

Comment:
isDarkThemeClicked this variable store boolean value so no need to check. == true you can directly put **isDarkThemeClicked.value** in if condition it is work as it is.